core/src/test/java/org/elasticsearch/tribe/TribeIT.java

```diff
@@ -554,17 +553,17 @@ private void putCustomMetaData(InternalTestCluster cluster, final TestCustomMeta
     private static void updateMetaData(InternalTestCluster cluster, UnaryOperator<MetaData.Builder> addCustoms) {
         ClusterService clusterService = cluster.getInstance(ClusterService.class, cluster.getMasterName());
         final CountDownLatch latch = new CountDownLatch(1);
-        clusterService.submitStateUpdateTask("update customMetaData", new ClusterStateUpdateTask(Priority.IMMEDIATE) {
+        clusterService.submitStateUpdateTask("update customMetaData", new LocalClusterUpdateTask(Priority.IMMEDIATE) {
```

Comment:
I'm not sure anymore (it might have had to do with the additional changes that I've made in an earlier version of this PR). I've changed it back.